File: app/templates/app-java/src/main/java/util/logging/CrashReportingTree.java
package <%= appPackage %>.util.logging;

import android.util.Log;
import timber.log.Timber;

public class CrashReportingTree extends Timber.Tree {

    @Override
    protected void log(int priority, String tag, String message, Throwable t) {
    if (priority == Log.VERBOSE || priority == Log.DEBUG) {
      return;
    }
  }
}

File: app/templates/app-kotlin/src/main/java/util/logging/CrashReportingTree.kt
package <%= appPackage %>.util.logging;

import android.util.Log;
import timber.log.Timber;

class CrashReportingTree : Timber.Tree() {

    override fun log(priority: Int, tag: String, message: String, t: Throwable) {
        if (priority == Log.VERBOSE || priority == Log.DEBUG) {
            return
        }
    }
}
